commit cc9cad401de313f81274910d5850f359a0d049d6
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-08-09T15:48:15+01:00 (8 weeks ago)
    
    TOOLS-2297 update deps/eng for copyright checks and motd work

diff --git a/deps/eng b/deps/eng
index 126c0f0..9058562 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit 126c0f032b8bfddd45b14e8ee14e73e9798a013f
+Subproject commit 9058562f2f59892795e470ce25443ff636c90478
